rxandroidble/src/main/java/com/polidea/rxandroidble2/exceptions/BleDisconnectedException.java

```diff
@@ -12,26 +15,58 @@
  */
 public class BleDisconnectedException extends BleException {
 
+    /**
+     * Set when the state is not available, for example when the adapter has been switched off.
+     */
+    public static final int UNKNOWN_STATUS = -1;
+
+    public static class BleAdapterDisabledException extends BleException {
+        // Disconnection related to disabled Bluetooth adapter
+    }
+
     @SuppressWarnings("WeakerAccess")
     @NonNull
     public final String bluetoothDeviceAddress;
+    public final int state;
 
-    @Deprecated
-    public BleDisconnectedException() {
```

Comment:
I know it is probably deprecated long enough but it is still a breaking API change